tensorflow/core/kernels/strided_slice_op.cc

```diff
@@ -28,14 +28,13 @@ limitations under the License.
 #include "tensorflow/core/framework/bounds_check.h"
 #include "tensorflow/core/framework/op_kernel.h"
 #include "tensorflow/core/framework/register_types.h"
+#include "tensorflow/core/framework/resource_var.h"
 #include "tensorflow/core/framework/tensor.h"
-#include "tensorflow/core/kernels/dense_update_functor.h"
```

Comment:
`tensorflow/core/kernels/strided_slice_op.cc:29:10: error: module //
/tensorflow/core/kernels:strided_slice_op does not depend on a module exporting '/tensorflow/core/framework/resource_var.h`